psi4/src/psi4/cc/cclambda/cclambda.cc

```diff
@@ -296,7 +295,25 @@ double CCLambdaWavefunction::compute_energy() {
             exit_io();
             throw PsiException("cclambda: error", __FILE__, __LINE__);
         }
-        if (pL_params[i].ground) overlap(pL_params[i].irrep);
+        if (pL_params[i].ground) {
+            auto LR_overlap = overlap(pL_params[i].irrep);
+            std::string gs_name; // What theory's lambda equations did we just solve?
+            if (params.wfn == "CC3" || params.wfn == "EOM_CC3") {
+                gs_name = "CC3";
+            } else if (params.wfn == "CC2" || params.wfn == "EOM_CC2") {
+                gs_name = "CC2";
+            } else if (params.wfn == "CCSD" || params.wfn == "EOM_CCSD" || params.wfn == "CCSD_AT") {
+                // In the CCSD_AT case, we solve the CCSD lambda equations so we can use them for a triples
+                // correction. For CCSD_AT analytic gradient theory, you would need to solve response equations
+                // including that triples correction, and only then would gs_name be CCSD_AT.
+                gs_name = "CCSD";
+            } else if (params.wfn == "CCSD_T") {
+                gs_name = "CCSD(T)";
+            } else {
+                throw PsiException("cclambda: unknown wfn", __FILE__, __LINE__);
+            }
+            reference_wavefunction_->set_scalar_variable("Left-Right " + gs_name + " Eigenvector Overlap", LR_overlap);
```

Comment:
Use all caps for var names for consistency, please. Lowercase are usually for placeholders like https://psicode.org/psi4manual/master/glossary_psivariables.html#psivar-CC-ROOT-n-CORRELATION-ENERGY

Also, non-all-caps likely messes up case insensitivity of `wfn.variable(SCF total energy)` lookup.

Reply:
oh, and add an entry in glossary.rst, please. I think the docs build will complain without.